index.js

```diff
@@ -13,20 +13,21 @@ const analyser                         = require('./analyser.js');
 const gui                              = require('./gui.js');
 
 // Dependencies, sorted in decending order of line length because it looks neater
-const winscreen                        = require('electron').remote.screen;
-const xml2js                           = require('xml2js').parseString;
-const { dialog }                       = require('electron').remote;
-const { shell, ipcRenderer, webFrame } = require('electron');
-const request                          = require('request');
-const semVer                           = require('semver');
-const path                             = require('path');
-const fs                               = require('fs');
+const winscreen											= require('electron').remote.screen;
+const xml2js											= require('xml2js').parseString;
+const { dialog, getCurrentWindow}						= require('electron').remote;
+const { shell, ipcRenderer, webFrame }					= require('electron');
+const request											= require('request');
+const semVer											= require('semver');
+const path												= require('path');
+const fs												= require('fs');

```

Comment:
why are the "=" not aligned and why are they so far out? Align them all one space after `const { shell, ipcRenderer, webFrame }` please.